docker/Dockerfile.arm

```
@@ -17,20 +31,57 @@ RUN vcpkgCommitId=$(grep '.builtin-baseline' vcpkg.json | awk -F: '{print $2}' |
 	&& git checkout $vcpkgCommitId \
 	&& ./bootstrap-vcpkg.sh
 
-WORKDIR /opt/vcpkg
-COPY vcpkg.json /opt/vcpkg/
-RUN VCPKG_FORCE_SYSTEM_BINARIES=1 /opt/vcpkg/vcpkg --feature-flags=binarycaching,manifests,versions install
+# Install dependencies using manifest mode standalone
+WORKDIR /opt/vcpkg_manifest
+COPY vcpkg.json /opt/vcpkg_manifest/
+
+	/bin/bash -euo pipefail -c '\
```

Hadolint (2.14.0)

[error] unexpected '/'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)